### DISCUSSION TITLE
guide/install/manual

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#4] [Announcements]
# guide/install/manual

Documentation for alist v3

https://alist.nn.ci/guide/install/manual.html

**Comment by yezhi233** [comment]
通过Daemon启动后，登录后台提示密码错误，./alist server方式启动正常

**Reply by duskNNNN** [reply]
因为进入后台需要密码，输入框中为空，与默认不匹配，肯定显示密码错误

**Comment by HurricanMaple** [comment]
replit部署方式已失效，运行下载alist仓库包命令就提示不可用，似乎已经屏蔽了这个存储库
